docs/synctl-create-test.md

```diff
@@ -44,21 +45,18 @@ synctl create test [options]
 ```
 ### Options for API Script test
 ```  
-    -f, --from-file <file>              load synthetic test payload from file (.json)
     --script <file>                     load script (.js) from file
     --bundle <bundle>                   Synthetic bundle test script, support zip file, zip file encoded with base64
```

Comment:
we can make it more clear like "Synthetic bundle test script, support zip file (.zip) path  or  zip file content encoded with base64